packages/babel-plugin-proposal-optional-chaining/test/fixtures/general/call/output.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+const foo = {
+  bar() {
+    return this;
+  }
+
+};
+(foo === null || foo === void 0 ? foo.bar() : foo.bar)();
```

Comment:
When `foo` is nullish, the conditional expression should return `undefined` instead of `foo.bar()`, otherwise we should return `foo.bar()`. One of transform results can be
```js
(foo === null || foo === void 0) ? (void 0)() : foo.bar()
```

Since optional chaining has been shipped in Chrome 80, one can also check the execution results between before and after.

```js
var foo = undefined
(foo?.bar)();
// TypeError: (intermediate value) is not a function
(foo === null || foo === void 0 ? foo.bar() : foo.bar)();
// TypeError: Cannot read property 'bar' of undefined
```

```js
var foo = {
 bar() { return this._bar },
 _bar: 42
}
(foo?.bar)();
// 42;
(foo === null || foo === void 0 ? foo.bar() : foo.bar)();
// undefined
```